tests/integration/path_test.py

```diff
@@ -5,6 +5,41 @@
 import pytest
 
 from gemdat.io import load_known_material
+from gemdat.path import optimal_path
+
+
+@pytest.vaspxml_available  # type: ignore
+def test_optimal_path(vasp_full_vol, vasp_F_graph):
```

Comment:
You could simplify this code by parametrizing it: https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids

This will make it easier to add new cases later on, and to identify which part failed.

Not necessary for this PR, just in case you are interested. For example (different project): https://github.com/duqtools/duqtools/blob/5d6bde0dd1506e0ce570e3dd3c37988ea52ca036/tests/ids/test_ids_operation.py#L147

Reply:
Thanks! I will leave this change out of the PR for now, but I am definitely interested in learning about parametrization